frontend/javascripts/oxalis/view/settings/dataset_settings_view.js

```diff
@@ -74,6 +74,23 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps> {
             <Tag style={{ cursor: "default", marginLeft: 8 }} color={isRGB && "#1890ff"}>
               {isRGB ? "24-bit" : "8-bit"} Layer
             </Tag>
+            {alpha > 0 && isVisible ? (
+              <Tooltip title="Make this color layer invisible.">
+                <Icon
+                  type="eye"
+                  onClick={() => this.props.onChangeLayer(layerName, "isVisible", false)}
+                  style={{ marginTop: 4, marginLeft: 8, cursor: "pointer" }}
+                />
+              </Tooltip>
+            ) : (
+              <Tooltip title="Make this color layer visible.">
```

Comment:
Let's combine this into one Tooltip:
  - Change the tooltip title to `${isVisible ? "Hide" : "Show"} layer.`
  - Change the icon type to `{isVisible ? "eye" : "eye-o"}`
  - Change the third parameter to `onChangeLayer` to `!isVisible`